packages/mui-material/src/Radio/Radio.js

```diff
@@ -235,10 +235,12 @@ Radio.propTypes /* remove-proptypes */ = {
   id: PropTypes.string,
   /**
    * [Attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Attributes) applied to the `input` element.
+   * @deprecated Use `slotProps.input` instead. This prop will be removed in v7. See [Migrating from deprecated APIs](/material-ui/migration/migrating-from-deprecated-apis/) for more details.
```

Comment:
@siriwatknp are changes in this file and in [this](https://github.com/mui/material-ui/pull/44974/files#diff-82ca9b8d368207e889963ad78d7c9dbbf88c1c4b67f751b24bf9d3d0c2512f9e) file are expected?

Reply:
Yes, this is automatically updated with the `SwitchBase` update.

Reply:
Maybe it's better if I include #44972 is this PR so that they are deprecated together.

Reply:
Oops, I've merged it. Shall I revert or will you merge other PRs